app/serializers/current_user_serializer.rb

```diff
@@ -122,6 +124,10 @@ def can_send_private_messages
     scope.can_send_private_messages?
   end
 
+  def use_admin_sidebar
+    object.admin? && object.in_any_groups?(SiteSetting.admin_sidebar_enabled_groups_map)
+  end
```

Comment:
```suggestion
  def use_admin_sidebar
    object.in_any_groups?(SiteSetting.admin_sidebar_enabled_groups_map)
  end

  def include_user_admin_sidebar?
    object.admin?
  end
```

Minor but there is no need to include the attribute for a user that is not an admin.

Reply:
Ah yeah good point!